ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/KeyStore.scala

```diff
@@ -108,3 +108,44 @@ class FileBasedKeyStoreBuilder(keyStoreType: String,
   }
 
 }
+
+class FileOnClasspathBasedKeyStoreBuilder(keyStoreType: String,
+                                          filePath: String,
+                                          password: Option[Array[Char]]) extends KeyStoreBuilder {
+
+  def build(): KeyStore = {
+
+    val is = getClass.getClassLoader.getResourceAsStream(filePath)
```

Comment:
@richdougherty will this work fine with Play's reloading or should we do something else?

Reply:
Yes, this is fine from Play's perspective (we do similar things internally). The only better alternative here considering Play is that if we could provide the ClassLoader instead.

Reply:
Thanks for confirming